app/models/cache/Answers.scala

```diff
@@ -57,6 +59,15 @@ case class ArrivalAnswers(
 
 object ArrivalAnswers {
   implicit val format: Format[ArrivalAnswers] = Json.format[ArrivalAnswers]
+
+  def apply(queryUcr: Option[UcrBlock]): ArrivalAnswers = queryUcr match {
+    case Some(ucrBlock) if ucrBlock.ucrType.equals(mucr) =>
+      ArrivalAnswers(consignmentReferences = Some(ConsignmentReferences(ConsignmentReferenceType.M, ucrBlock.ucr)))
```

Comment:
Could you please use parameter names for `ConsignmentReferences` ?

Reply:
You mean `ConsignmentReferences(reference = ConsignmentReferenceType.M, referenceValue = ucrBlock.ucr))` ?  Is that necessary for clarity?

Reply:
I think it's not necessary in cases where we're filling all arguments and additionally those arguments are really good described. I mean `ConsignmentReferenceType.M` `ucrBlock.ucr`

Reply:
Why you're not using `UcrBlock.ducr`, but `ConsignmentReferenceType.D` ?

Reply:
Don't understand . 
UcrBlock.mucr => ConsignmentReferenceType.M
UcrBlock.ducr => ConsignmentReferenceType.D

Reply:
My fault, misunderstood. It's okay. I thought that UcrBlock.mucr has the same value like ConsignmentReferenceType.M